Frends.Community.Azure.Blob/UploadTask.cs

```diff
@@ -54,16 +55,22 @@ public class UploadTask
             else
                 fileName = destinationProperties.RenameTo;
 
+            Dictionary<string, string> tags = new Dictionary<string, string>();
+            if (input.Tags != null)
+                if (input.Tags.Length > 0)
```

Comment:
Could these 2 conditions be combined?

Reply:
Actually no. If they would be in the same if-expression, like `if (input.Tags != null && input.Tags.Length > 0)` then both of those expressions would be evaluated at the same time. If `input.Tags` is null, then `input.Tags.Length > 0` would fail with `NullPointerException`. Hence, we gotta first check if the variable is not null before we can check if there is any objects in the array.